src/Task/Brunch.php

```diff
@@ -13,18 +14,15 @@
  */
 class Brunch extends AbstractExternalTask
 {
-    /**
-     * @return string
-     */
-    public function getName()
+    public function getName(): string
     {
         return 'brunch';
     }
 
     /**
      * @return OptionsResolver
```

Comment:
This docblock can be removed


(Note: This is on every task - I wont copy paste that remark again :) )